docs/_src/language/basic.md

```diff
@@ -145,7 +145,7 @@ query: airports->{
 
 In Malloy, ordering and limiting work pretty much the same way they do in SQL, though Malloy introduces some [reasonable defaults](order_by.md).
 
-The `top:` and `limit:` statements are synonyms and limits the number of rows returned. Results below are sorted by the first measure decending—in this case, `airport_count`.
+The `top:` and `limit:` statements are synonyms and limits the number of rows returned. Results below are sorted by the first measure descending--in this case, `airport_count`.
```

Comment:
I think we should use `—` instead of `--`.

Reply:
ah thought this was a hyphen